packages/form-js-editor/test/spec/properties-panel/PropertiesPanel.spec.js

```diff
@@ -575,6 +580,108 @@ describe('properties panel', function() {
 
       });
 
+
+      describe('dynamic values', function() {

```

Comment:
Given the current test structure, I think it's fine to have these tests in the `PropertiesPanel.spec.js`. In a perfect world, we would maybe have them in separate files, similar to how we do it in BPMN land.